src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -734,6 +736,45 @@ def _test_data_encryption(self, resource_group, server, database_engine, vault_n
                  .format(database_engine, resource_group, server),
                  checks=[JMESPathCheck('length(@)', 0)])
 
+    def _test_aad_admin(self, resource_group, server, database_engine):
+        oid = '5e90ef3b-9b42-4777-819b-25c36961ea4d'
+        oid2 = 'e4d43337-d52c-4a0c-b581-09055e0359a0'
+        user = 'DSEngAll'
+        user2 = 'TestUser'
+
+        self.cmd('{} server ad-admin create --name {} -g {} -i {} -u {}'
```

Comment:
Please add test for creating with --no-wait parameter and `wait` command.
Out of curiosity, how long does it take for the create operation?

Reply:
at max 10 minutes.